taskchampion/xtask/src/main.rs

```diff
@@ -3,17 +3,42 @@
 //! At the moment it is very simple, but if this grows more subcommands then
 //! it will be sensible to use `clap` or another similar library.
 
+//! The const MSRV_FILE_PATHS_REGEX in /xtask/main.rs is an array of tuples
```

Comment:
```suggestion
//! The const MSRV_PATHS_REGEX in /xtask/main.rs is an array of tuples
```
you could also move this rustdoc comment down a few lines to document the `MSRV_PATH_REGEX` itself.